formspree/routes.py

```diff
@@ -37,18 +38,18 @@ def configure_routes(app):
     app.add_url_rule('/logout', 'logout', view_func=uv.logout, methods=['GET'])
 
     # Users' forms
-    app.add_url_rule('/dashboard', 'dashboard', view_func=fv.forms, methods=['GET'])
-    app.add_url_rule('/forms', 'forms', view_func=fv.forms, methods=['GET'])
-    app.add_url_rule('/forms', 'create-form', view_func=fv.create_form, methods=['POST'])
-    app.add_url_rule('/forms/sitewide-check', view_func=fv.sitewide_check, methods=['GET'])
-    app.add_url_rule('/forms/<hashid>/', 'form-submissions', view_func=fv.form_submissions, methods=['GET'])
-    app.add_url_rule('/forms/<hashid>.<format>', 'form-submissions', view_func=fv.form_submissions, methods=['GET'])
-    app.add_url_rule('/forms/<hashid>/toggle-recaptcha', 'toggle-recaptcha', view_func=fv.form_recaptcha_toggle, methods=['POST'])
-    app.add_url_rule('/forms/<hashid>/toggle-emails', 'toggle-emails', view_func=fv.form_email_notification_toggle, methods=['POST'])
-    app.add_url_rule('/forms/<hashid>/toggle-storage', 'toggle-storage', view_func=fv.form_archive_toggle, methods=['POST'])
-    app.add_url_rule('/forms/<hashid>/toggle', 'form-toggle', view_func=fv.form_toggle, methods=['POST'])
-    app.add_url_rule('/forms/<hashid>/delete', 'form-deletion', view_func=fv.form_deletion, methods=['POST'])
-    app.add_url_rule('/forms/<hashid>/delete/<submissionid>', 'submission-deletion', view_func=fv.submission_deletion, methods=['POST'])
+    app.add_url_rule('/dashboard', 'dashboard', view_func=fv.serve_dashboard, methods=['GET'])
+    app.add_url_rule('/forms', 'dashboard', view_func=fv.serve_dashboard, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>', view_func=fv.serve_dashboard, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>/<path:s>', view_func=fv.serve_dashboard, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>.<format>', view_func=fv.export_submissions, methods=['GET'])
+    app.add_url_rule('/api/forms', view_func=fa.list, methods=['GET'])
```

Comment:
If this will be a public API, I think we should version it from the start. Something like `/api/v1/forms`. Makes things cleaner if we need to later fork and create a new version of the api.

Reply:
The idea (from our previous conversations) was that it wouldn't be public.

Well, I guess all endpoints we have are "public", but we don't tell anyone to use them (and they use cookie-based authentication) so we can change them and not tell anyone about it, because they shouldn't be using it anyway.

Reply:
Ok. I didn't remember where we landed there. If these are internal APIs then sure. However I think we might want to save the /api path for public apis. Can we call it something else like /api-int/ ?

Reply:
Yes, that's a good idea.